src/components/Group/GroupEdit.vue

```diff
@@ -4,14 +4,15 @@
     <h3 v-else><i class="fa fa-edit"></i> {{ $t('GROUP.EDIT') }}</h3>
     <q-card>
       <div class="edit" :class="{ changed: hasChanged }">
-        <form @submit="save">
+        <form @submit.prevent="save">
```

Comment:
Ah good, I noticed this was doing a full page submit recently :)

Reply:
I think it's needed in more places, will add an issue :)